devtools/package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "preact-detools",
+	"amdName": "preactDevtools",
+	"version": "1.0.0",
+	"private": true,
+	"description": "Preact bridge for Preact devtools",
+	"main": "dist/devtools.js",
+	"module": "dist/devtools.module.js",
+	"umd:main": "dist/devtools.umd.js",
+	"source": "src/index.js",
+	"license": "MIT",
+	"mangle": {
```

Comment:
Isn't mangle already defined in `mangle.json`?

Reply:
Good catch :+1: